addons/prometheus/template/generate.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -euo pipefail
+
+VERSION=""
+function get_latest_release_version() {
+    VERSION=$(curl -I https://github.com/prometheus-operator/prometheus-operator/releases/latest | \
+        grep -i "^location" | \
+        grep -Eo "[0-9]\.[0-9]+\.[0-9]+")
+}
+
+function generate() {
+    # make the base set of files
+    mkdir -p "../${VERSION}"
+    cp -r ./base/* "../${VERSION}"
+    mkdir -p "../${VERSION}/crds"
+    mkdir -p "../${VERSION}/operator"
+
+    # run the helm commands
+    helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    helm repo update
+
+    # get a copy of the stack with CRDs
+    helm template replaceme prometheus-community/kube-prometheus-stack --values ./values.yaml -n monitoring --include-crds > "../$VERSION/crds/crds-all.yaml"
```

Comment:
should you add --version arg here?

Reply:
would we ever want a version other than latest?

Reply:
well you were lifting the version from github but using the helm chart

Reply:
ah. well, now they're both using latest :)